### DISCUSSION TITLE
have two projects (Laravel & Next.js) on the same top domain

### DISCUSSION THREAD

**Discussion by NAB-TAG** [vercel/community#5791] [Help] [answered]
### Summary

Is it possible to have two different projects (Laravel & Next.js) on the same top domain?

For example, have the Laravel project in api.example.vercel.app
and have a project in next.js in example.vercel.app

### Steps to Reproduce

If it is possible to put an app in Laravel

This doubt occurs because to pass session cookies it is necessary to have the same top domain.

**Comment by amyegan** [comment, marked as answer]
Hey @NAB-TAG! There is a [community runtime](https://vercel.com/docs/functions/runtimes#community-runtimes) for PHP. More details on that are available in the repo at https://github.com/vercel-community/php if you need help or run into issues with it.

And you can use [rewrites](https://vercel.com/docs/edge-network/rewrites) on Vercel to [serve multiple projects under a single domain](https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain). 🙂
